Index: class06/src/Code04_CopyListWithRandom.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/class06/src/Code04_CopyListWithRandom.java b/class06/src/Code04_CopyListWithRandom.java
new file mode 100644
--- /dev/null	(date 1659794755715)
+++ b/class06/src/Code04_CopyListWithRandom.java	(date 1659794755715)
@@ -0,0 +1,7 @@
+/**
+ * @author : WXY
+ * @create : 2022-08-06 22:02
+ * @Info :
+ */
+public class Code04_CopyListWithRandom {
+}
